spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+  describe "GET /index" do
+    it "returns http success" do
+      get "/users/"
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('index of users')
+    end
+  end
+
+  describe "GET /show" do
+    it "returns http success" do
+      get "/users/show"
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('show user')
+    end
+  end
+end
```

Comment:
- [ ] Good job so far, but you should test if a correct template was rendered for `show` and `index` actions like `expect(response).to render_template(:show)`. Please kindly add them 🙃 🙃

Reply:
Oh, I will add this specs in a while, thank you